bip-DLEQ.mediawiki

```diff
@@ -0,0 +1,90 @@
+<pre>
+  BIP: ?
+  Title: Discrete Log Equality Proofs over secp256k1
```

Comment:
Consider adding `Layer: Applications` above this line.

Reply:
Added.